python/sqlcommenter-python/google/cloud/sqlcommenter/__init__.py

```diff
@@ -26,7 +26,7 @@
 KEY_VALUE_DELIMITER = ','
 
 
-def generate_sql_comment(**meta):
+def generate_sql_comment(sql, **meta):
```

Comment:
The name is confusing. It's not **only** generating sql comment but also mutating the query. 

Moreover, i am thinking we should not pass the sql query here and do mutation.

The reason is we should avoid doing any custom operation as much as possible. As django does not have the issue with semicolon, we will just append the comment there instead of stripping spaces, checking `;`, etc. 

Instead we can have a separate function `add_sql_comment` i.e.

```
def generate_sql_comment(**meta):
  ...

def add_sql_comment(sql, **meta):
  comment = generate_sql_comment(meta)
  sql = sql.rstrip()
  ...


```

Reply:
Created add_sql_comment and add the suggested change
Django has the ability to execute raw query. 
https://stackoverflow.com/questions/5931586/raw-sql-queries-in-django-views

This PR works good if semicolon is appended in the raw query